src/libraries/Microsoft.NET.WebAssembly.Webcil/src/Webcil/WebcilReader.cs

```diff
@@ -240,6 +240,17 @@ private static CodeViewDebugDirectoryData MakeCodeViewDebugDirectoryData(Guid gu
         return (CodeViewDebugDirectoryData)mi.Invoke(new object[] { guid, age, path });
     }
 
+    private static PdbChecksumDebugDirectoryData MakePdbChecksumDebugDirectoryData(string algorithmName, ImmutableArray<byte> checksum)
+    {
+        var types = new Type[] { typeof(string), typeof(ImmutableArray<byte>) };
+        var mi = typeof(PdbChecksumDebugDirectoryData).GetConstructor(BindingFlags.NonPublic | BindingFlags.Instance, null, types, null);
```

Comment:
Would it make sense to cache this, if this is getting called per assembly?

Reply:
yea probably :D

Reply:
Added some lazy helpers